### DISCUSSION TITLE
What's a task.ms link?

### DISCUSSION THREAD

**Discussion by BenJKuhn** [microsoft/WindowsAppSDK#1977] [General] [answered]
This is more of an answer than a question. A few people have asked me & my teammates about task.ms links from GitHub projects to a Microsoft private resource. They link to a non-public ADO database. We use these links for a few different reasons:

1) Some of our contributing teams schedule work across numerous projects and prefer to track all of their work against a single database or view. They cross-link to their work database so that they can coordinate work across this and other projects in one place.

2) We sometimes work closely with partners inside or outside Microsoft to address an issue. We may be given helpful context or diagnostics from a partner that they would prefer we not disclose publicly. We use our internal database to associate the details with the GitHub issue without disclosing the partner's information.

We'll do our best keep engineering details in the open, but sometimes we need to bridge to the "closed" world, and this is our tool for doing that.

Thanks,

Ben

**Comment by btueffers** [comment, marked as answer]
Thanks @BenJKuhn!
